xyz

```diff
@@ -34,13 +35,17 @@ Options:
    --prerelease-label <label>
         Specify the label to be used in the version number when publishing
         a pre-release version (e.g. 'beta' is the label in '2.0.0-beta.0').
-        'rc' is assumed if this option is omitted.
+        'rc' is assumed if this option is omitted. If the release is a
+        prerelease, as indicated by --increment, the --prerelease-label will
```

Comment:
We're using *pre-release* (hyphenated) two lines above. To be consistent, let's hyphenate the word in contexts in which it does not have special meaning.